src/sentry/celery.py

```diff
@@ -51,6 +51,12 @@ def _validate_thread_sharing(self):
 
 patch_thread_ident()
 
+# This monkey patches the django 1.8 migration executor with a backported 1.9 executor.
```

Comment:
If possible I'd like to find a more permanent place for mocking Django/Django migrations. We're going to have some permanent monkeypatches to carry over some functionality that we implemented in South.

I wonder if this would make sense to import in `initialize_app`?

Reply:
Sure, I just slapped it here initially since that was blocking me from moving forward with evaluating performance. I'll try that.